addons/hr/models/hr_department.py

```diff
@@ -99,6 +99,8 @@ def write(self, vals):
                 # subscribe the manager user
                 if manager.user_id:
                     self.message_subscribe(partner_ids=manager.user_id.partner_id.ids)
+            if self.manager_id and self.manager_id.user_id:
```

Comment:
You should filter the records for which the manager actually changed. Otherwise you'll subscribe him anyway.

```suggestion
            for department in self:
                if department.manager_id.user_id and department.manager_id != manager:
                    department.message_unsubscribe(partner_ids=department.manager_id.user_id.partner_id.ids)
```

Reply:
yep yep I will corretc it